spec/export/index.md

```diff
@@ -298,38 +313,91 @@ Content-Type: application/json
-
-  - ```request``` - the full URL of the original bulk data kick-off request
-
-  - ```requiresAccessToken``` - boolean value of ```true``` or ```false``` indicating whether downloading the generated files requires a bearer access token.
-
-    Value SHALL be ```true``` if both the file server and the FHIR API server control access using OAuth 2.0 bearer tokens. Value MAY be ```false``` for file servers that use access-control schemes other than OAuth 2.0, such as downloads from Amazon S3 bucket URLs or verifiable file servers within an organization's firewall.
-
-  - ```output``` - array of file items with one entry for each generated file. If no resources are returned from the kick-off request, the server SHOULD return an empty array.
-
-    Each file item SHALL contain the following fields:
-
-      - ```type``` - the FHIR resource type that is contained in the file.
-
-        Each file SHALL contain resources of only one type, but a server MAY create more than one file for each resource type returned. The number of resources contained in a file MAY vary between servers. If no data are found for a resource, the server SHOULD NOT return an output item for that resource in the response. These rules apply only to top-level resources within the response; as always in FHIR, any resource MAY have a "contained" array that includes referenced resources of other types.
-
-      - ```url``` - the path to the file. The format of the file SHOULD reflect that requested in the ```_outputFormat``` parameter of the initial kick-off request.
-
-    Each file item MAY optionally contain the following field:
-
-      - ```count``` - the number of resources in the file, represented as a JSON number.  
-
-
-  - ```error``` - array of error file items following the same structure as the `output` array.
-
-    Errors that occurred during the export should only be included here (not in output). If no errors occurred, the server SHOULD return an empty array.  Only the `OperationOutcome` resource type is currently supported, so a server SHALL generate files in the same format as bulk data output files that contain `OperationOutcome` resources.
-
-The response body and any file item MAY optionally contain the following field:
-
-   - ```extension``` - To support extensions, this implementation guide reserves the name ```extension``` and will never define a field with that name, allowing server implementations to use it to provide custom behavior and information. For example, a server may choose to provide a custom extension that contains a decryption key for encrypted ndjson files. The value of an extension element SHALL be a pre-coordinated JSON object.
+<table class="table">
+  <thead>
+    <th>Parameters</th>
+    <th>Optionality</th>
+    <th>Type</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>alg</code></td>
```

Comment:
What is this? Is this `<tr>` a copy/paste error?

Reply:
yup thanks for catching that. fixed here https://github.com/HL7/bulk-data/pull/34/commits/810c7fa9f769f0fb3b16162dbc6e343062141a83